zetaclient/evm/evm_signer.go

```diff
@@ -33,20 +33,19 @@ import (
 )
 
 type Signer struct {
```

Comment:
Can we add comment on this structure?

Reply:
https://github.com/zeta-chain/node/pull/1890/commits/1321dccf1e8bc841a1d84e04ae3ea9140ad2ed74